pkg/chunked/storage_linux.go

```
@@ -1176,6 +1202,10 @@ func (c *chunkedDiffer) copyAllBlobToFile(destination *os.File) (digest.Digest,
 	// copy the entire tarball and compute its digest
 	_, err = io.CopyBuffer(destination, r, c.copyBuffer)
```

Aren't we ignoring this error now?